operator/src/main/java/dev/responsive/k8s/operator/reconciler/DemoPolicyPlugin.java

```diff
@@ -104,89 +117,43 @@ public void reconcile(
           .withName(appName)
           .edit(d -> {
             if (d.getMetadata().getResourceVersion()
-                .equals(deployment.getMetadata().getResourceVersion())) {
+                .equals(managedApp.getResourceVersion())) {
               d.getSpec().setReplicas(targetReplicas);
```

Comment:
this is the call that actually scales the replicas up or down - you'd need to update this to use the right client as well (right now its the `deployment` client)

Reply:
Ah. Yes, I missed that. Good catch, thanks!